BraveRewardsUI/Tipping/TippingViewController.swift

```diff
@@ -63,6 +72,73 @@ class TippingViewController: UIViewController, UIViewControllerTransitioningDele
     tippingView.optionSelectionView.options = [1.0, 5.0, 10.0].map {
       TippingOption.batAmount(BATValue($0), dollarValue: state.ledger.dollarStringForBATAmount($0) ?? "")
     }
+    
+    loadPublisherBanner()
+  }
+  
+  // MARK: - Networking
+  private func loadPublisherBanner() {
+    state.ledger.publisherBanner(forId: self.publisherInfo.id) { [weak self] banner in
+      guard let self = self, let banner = banner else { return }
+      
+      self.tippingView.overviewView.titleLabel.text = banner.title.isEmpty ? Strings.TippingOverviewTitle : banner.title
+      self.tippingView.overviewView.bodyLabel.text = banner.desc.isEmpty ? Strings.TippingOverviewBody : banner.desc
+      
+      self.downloadImage(url: banner.background, { image in
+        self.tippingView.overviewView.headerView.image = image
+      })
+      
+      self.downloadImage(url: self.publisherInfo.faviconUrl, { image in
```

Comment:
Does this work for you, does it download correct favicon?

In other places we use a delegate method to retrieve favicon `state.dataSource?.retrieveFavicon(with: faviconURL, completion: {[weak self] favData in`
cc @kylehickinson for visibility

Reply:
Fixed.